definitions/variable/industry/production.yaml

```diff
@@ -0,0 +1,19 @@
+- Production|{Industry Sector} [{Production Unit}]: # ???: do we need this aggregate?
+    description: Production of {Industry Sector} in {Production Unit}
+    unit: Mt/yr
+
+- Production|Iron and Steel|{Iron Commodity} [{Production Unit}]:
+    description: Production of {Iron Commodity} in {Production Unit}
+    unit: Mt/yr
```

Comment:
You use the tag-list for "Production Unit", but the "Value"-variable should not have the unit *Mt/yr* but *USD_2010/yr*. So either you also set the units depending on the tag-list or you only create the Volumne-variable-variant in this PR.